randovania/gui/auto_tracker_window.py

```diff
@@ -153,5 +196,7 @@ def selected_builder(self) -> ConnectorBuilder | None:
     def on_game_state_updated(self, state: ConnectedGameState):
         self.create_tracker()
         expected_connector = self.game_connection.get_connector_for_builder(self.selected_builder())
-        if expected_connector == state.source:
+        if expected_connector == state.source or self._last_source == state.source:
+            self._last_source = state.source
             self.item_tracker.update_state(state.current_inventory)
+            self.connected_game_state_label.setText(f"{state.source.game_enum.long_name}: {state.status.pretty_text}")
```

Comment:
That's missig the update if you open the auto tracker after the connector updated the status and will say only `Disconnected`.
Maybe read it once on open + changing the selected connector? Cause I was easily a able to get these wrong results:
Disconnected but I'm on title screen:
![image](https://github.com/randovania/randovania/assets/117127188/9041cc47-2dd8-4cd9-858d-867c8b82282a)
Echoes showing status of Dread:
![image](https://github.com/randovania/randovania/assets/117127188/9cb1129f-5b17-4bab-89a5-229176971e9e)

Reply:
Addressed, with a bunch other weird cases too

Reply:
Has still some unexpected behaviour for me. Like switching from a connected dread to an unconnected dolphin, stays on the dread tracker and an empty connection state label. Don't know if someone will every do this but at least I want to show it:
![image](https://github.com/randovania/randovania/assets/117127188/b80616d2-b99a-41c2-8e7f-62acee523bd4)

Reply:
Ah it's intentional to keep the tracker: if you go from non-disconnected to disconnected, it keeps the tracker in case it was a temporary connection loss.
The lack of any sort of label below is weird, but that's the message I put when the builder has no connector associated. Maybe it should say "Not Connected"

Reply:
Keeping the tracker makes only sense to me if the game connection was not changed or better to say: Discard the old state if some is fiddling around in the selector box.

Reply:
Made the tracker reset if changing builders